package.json

```diff
@@ -30,7 +30,7 @@
     "b4a": "^1.3.1",
     "blake2b-universal": "^1.0.1",
     "derive-key": "^1.0.1",
-    "hypercore": "next",
+    "hypercore": "github:hypercore-protocol/hypercore-next#abstract-auth",
```

Comment:
Everything great except this -- let's just keep keep this set to `next` (since the @mafintosh just alpha released the auth changes on `next`).

Reply:
Ok just removed that commit